ini/stm32f4.ini

```diff
@@ -730,7 +730,7 @@ upload_protocol = stlink
 [env:STM32F401RC_btt]
 extends                     = stm32_variant
 platform                    = ststm32@~14.1.0
-platform_packages           = framework-arduinoststm32@https://github.com/stm32duino/Arduino_Core_STM32/archive/main.zip
+platform_packages           = framework-arduinoststm32@https://github.com/stm32duino/Arduino_Core_STM32/archive/6f39b418e53bf827650399feacda9c3d3b245b57.zip
```

Comment:
Since `Arduino_Core_STM32` actually does releases, I think it'd be better to use them instead of commit: https://github.com/stm32duino/Arduino_Core_STM32/releases